feat: allow list of client ids or config names in LspRestart and LspStop (#3438)
neovim/nvim-lspconfig: README.md

* `:LspInfo` (deprecated alias to `:che lspconfig`) shows the status of active and configured language servers.
* `:LspStart <config_name>` Start the requested server name. Will only successfully start if the command detects a root directory matching the current config. Pass `autostart = false` to your `.setup{}` call for a language server if you would like to launch clients solely with this command. Defaults to all servers matching current buffer filetype.

> * `:LspStop {<client_id> or <config_name>} ...` Stops the servers with the given client-ids or config names. Defaults to
> stopping all servers active on the current buffer. To force stop add `++force`
> * `:LspRestart {<client_id> or <config_name>} ...` Restarts the clients with the given client-ids or config names, and attempts to reattach to all

Reviewer: ```suggestion
* `:LspStop <client_id_or_name> [<client_id_or_name> ...]` Stops the given server(s). Defaults to
stopping all servers active on the current buffer. To force stop add `++force`
* `:LspRestart <client_id_or_name> [<client_id_or_name> ...]` Restarts the given client(s), and attempts to reattach to all
```
